docs/architecture/modern-web-apps-azure/common-client-side-web-technologies.md

```diff
@@ -162,7 +162,7 @@ Running .NET code inside web browsers is made possible by WebAssembly (abbreviat
 
 WebAssembly code can access the full functionality of the browser via JavaScript, called JavaScript interoperability, often shortened to JavaScript interop or JS interop. .NET code executed via WebAssembly in the browser runs in the browser's JavaScript sandbox with the protections that the sandbox provides against malicious actions on the client machine.
 
-For more details, see [Introduction to ASP.NET Core Blazor](https://docs.microsoft.com/aspnet/core/blazor/?view=aspnetcore-5.0)
+For more details, see [Introduction to ASP.NET Core Blazor](/aspnet/core/blazor/?view=aspnetcore-5.0)
```

Comment:
The phrase "more details" seems awkward to me. I recommend our standard approach ...

```suggestion
For more information, see [Introduction to ASP.NET Core Blazor](/aspnet/core/blazor/).
```